pkg/operator/api/api.go

```diff
@@ -512,41 +524,42 @@ func (a *apiServer) HTTPEndpointUpdate(in *operatorv1pb.HTTPEndpointUpdateReques
 	key := keyObj.String()
 
 	a.endpointLock.Lock()
-	a.allEndpointsUpdateChan[key] = make(chan *httpendpointsapi.HTTPEndpoint, 1)
-	updateChan := a.allEndpointsUpdateChan[key]
+	a.endpointsUpdateChan[key] = make(chan *loader.Event[httpendpointsapi.HTTPEndpoint], 10)
```

Comment:
Another magic number for capacity. Why 10?

Reply:
Changed.